tests/server/admin-api/v1/apiBackends.js

```diff
@@ -14,6 +14,12 @@ Tests for the API Backends v1 wrapper.
 Tinytest.add(
   "Admin API v1 - API Backends - getBackends",
   function (test) {
+    var config = {
+      api_key: Meteor.settings.apiUmbrella.api_key,
```

Comment:
Again, using camel case now, to follow common JS practice. I.e. `apiKey`, `authToken`, `baseUrl`.